### ISSUE TITLE
Using with typescript

### ISSUE THREAD

**Issue by threeaccents** [thanhtunguet/grapesjs-react#27]
When i try to load this library in a create-react-app typescript project I get the following error:

`Could not find a declaration file for module 'grapesjs-react'. `

I tried installing the @types but it does not exist. I ended up declaring a dummy `.d.ts` file but having one included in the project would be nice. 

Thanks for the package!

**Comment by DigiPascal** [comment]
Hi @threeaccents,

"I ended up declaring a dummy .d.ts file" > Can you describe what you did ? 
I have the same issue and I have no clue how to fix this...

Thanks

**Comment by threeaccents** [comment]
@DigiPascal create a file named `grapejs.d.ts` inside of it add the following:

```typescript
declare module 'grapesjs-react';
```
Then in what ever file you're gonna use the grapejs component add at the top:
`/// <reference path="path/to/grape.d.ts" />`

And then you should be good to go. You won't get any types while using the component but at least the compiler won't complain.

**Comment by thanhtunguet** [comment]
Hi all,
I am using rollup to compile this package but I don't know how to config rollup to generate d.ts file, can anyone help me?

**Comment by threeaccents** [comment]
I was checking out the code more and I noticed the declaration file is already being generated in the dist folder as `grapesjs-react.d.ts` so I think all you need to do is include it in the `package.json`

```json
{
   ....
  "main": "./dist/grapesjs-react.cjs.js",
  "module": "./dist/grapesjs-react.js",
  "types": "./dist/grapesjs-react.d.ts",
}
```

Here is a PR https://github.com/thanhtunguet/grapesjs-react/pull/30

**Comment by threeaccents** [comment]
@DigiPascal If you want proper types until the PR is merged just go into your `node_modules/grapesjs-react/package.json`
 
and add 
`"types": "./dist/grapesjs-react.d.ts",`

You can then remove the dummy declarations file I had recommended earlier. Will probably need to restart the server to see it work and your code editor as well.

**Comment by thanhtunguet** [comment]
> @DigiPascal If you want proper types until the PR is merged just go into your `node_modules/grapesjs-react/package.json`
> 
> and add
> `"types": "./dist/grapesjs-react.d.ts",`
> 
> You can then remove the dummy declarations file I had recommended earlier. Will probably need to restart the server to see it work and your code editor as well.

Thank you. I have updated package.json file and then published v3.0.4. Please check it out

**Comment by DigiPascal** [comment]
Tried it from scratch and works like a charm! :-)

Thanks @threeaccents and @thanhtunguet
